masterlist.yaml

```diff
@@ -151,7 +150,9 @@ plugins:
     msg:
       - type: warn
         content: 'You need to check your Tale of Two Wastelands load order'
-        condition: 'file("TaleOfTwoWastelands.esm") and not checksum("TaleOfTwoWastelands.esm", 26497E29)'
+        condition: 'file("TaleOfTwoWastelands.esm") and not checksum("TaleOfTwoWastelands.esm", 6D6F2EA7)'
```

Comment:
As of version 2.8.1a version in TaleOfTwoWastelands.esm was left untouched, so checksum, for now, is more preferred as it seems.

Point of this warning is to point users to check load order, if they have not latest version of TTW, as TTW project in alpha state by this moment.